ecs-init/config/common.go

```diff
@@ -29,8 +31,18 @@ const (
 	AgentLogFile = "ecs-agent.log"
 
 	UnixSocketPrefix = "unix://"
+
+	// Default region name
+	DefaultRegionName = "default"
 )
 
+var S3BucketMap = map[string]string{
```

Comment:
Few things here.

1. `S3BucketMap` is global scoped, it doesn't need to be 
2. Please make this a prefix map so that it constructs `"default"` -> `"https://s3.amazonaws.com/amazon-ecs-agent/"` and move `ecs-agent-v1.14.1.tar` to be a const so that the map can be used to get the prefix and the latest tarball is only typed once some where
3. Please remove entries for `"us-east-1"` and `"us-west-2"` from this map.